core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -700,6 +758,58 @@ oppia.factory('explorationStatesService', [
         return (
           validatorsService.isValidStateName(newStateName, showWarnings));
       },
+      getStateContentMemento: function(stateName) {
+        return getStatePropertyMemento(stateName, 'content');
+      },
+      saveStateContent: function(stateName, newContent) {
+        saveStateProperty(stateName, 'content', newContent);
+      },
+      getStateParamChangesMemento: function(stateName) {
+        return getStatePropertyMemento(stateName, 'param_changes');
+      },
+      saveStateParamChanges: function(stateName, newParamChanges) {
```

Comment:
Are these 'save' functions covered by tests? I can't imagine it would be too hard to add some copy and paste tests to ensure the before/after/memento behavior works for all these setters/getters and would be nice. :)